### ISSUE TITLE
Contributions are open! #Hacktoberfest

### ISSUE THREAD

**Issue by sakshamsingh01** [SakshamSingh-v2/First-Projects#1]
Make sure to read the readme.md file for all instructions for contribution.
All contributions are welcome

- [ ] Code

- [ ] Documentation

- [ ] New feature

**Let's Celebrate the Hacktoberfest 2020 and help the beginners!**

**Comment by abdullah2205** [comment]
Hi I am new to github, can i contribute to this project?

**Comment by Vrindagupta6828** [comment]
@SakshamSingh-v2 I want to work on this. please assign.

**Comment by sakshamsingh01** [comment]
> Hi I am new to github, can i contribute to this project?

Yes sure @abdullah2205! This project is all for beginners. Just make sure to read the README and CONTRIBUTION files for instructions.

**Comment by kazimsayed954** [comment]
I want to work on this. please assign.

**Comment by sakshamsingh01** [comment]
@kazimsayed954 You can work on this. As this repo includes the first projects contributions from anyone, this can not be assigned to one single person. 
Go ahead! Read the readme.md and start contributing. :-)

**Comment by Vrindagupta6828** [comment]
@sakshamsingh-v2 please review my work

**Comment by sakshamsingh01** [comment]
Great work @Vrindagupta6828 ! I have merged your PR. Now just make sure to add your name in the CONTRIBUTORS.md file present in root folder.

**Comment by Vrindagupta6828** [comment]
@SakshamSingh-v2 done and thanks

**Comment by Vansh1067** [comment]
I want to work on this. please assign

**Comment by sakshamsingh01** [comment]
If you want to contribute in Documentation #36 #35 or #13 Then I can assign.
If you want to contribute by uploading any project, then go on! @Vansh1067

**Comment by AKASH-18** [comment]
Can I add my js projects to it.

**Comment by sakshamsingh01** [comment]
Ya sure @AKASH-18

**Comment by Vansh1067** [comment]
Can I add my html ,css and js project to it

**Comment by sakshamsingh01** [comment]
Yes sure @Vansh1067 . Make sure to follow the contribution guidelines

**Comment by sakshamsingh01** [comment]
Hacktoberfest 2021 is here. The contributions are open!

**Comment by INVOKER2002** [comment]
hey saksham , will contributing to this repo add to my hacktoberfest contribution as well ?
